### ISSUE TITLE
Create kernelspec with different base image

### ISSUE THREAD

**Issue by barche** [JuliaCloud/JuliaBox#475]
I am trying to create a JuliaBox installation that only has the latest 0.4 and 0.5 stable versions in the kernel list, but instead end up with no available kernels at all (only Python). The changes made to the dockerfiles and install scripts are here:
https://github.com/barche/JuliaBox/commit/e0f79e10d1b0ecacdacb898740be297cd99f79bd

It seems that if not using the 0.3 base image, the kernelspecs are written to a directory in `.local`. I have attempted to correct this, but no luck so far.

**Comment by tanmaykm** [comment]
I think the kernel list gets built while building the package bundles and are loaded from a separate volume mount (after #421). Can you rebuild the package bundles and check?

**Comment by barche** [comment]
Yes, they are, but somehow when not using a 0.3 base image they are not put in `.ipython` anymore. I think there is a newer IJulia on 0.4 and higher that uses jupyter instead of ipython. I adapted the config following http://jupyter.readthedocs.io/en/latest/migrating.html and got a little further.

In my config-rma branch the kernels show now, but the 0.5 kernel dies:
https://github.com/barche/JuliaBox/tree/config-rma

Is there a log or something that shows the reason for a dying kernel?

**Comment by barche** [comment]
Never mind, I found the log at `/opt/juliabox/.juliabox/ijulia_err.log` and can read it using the shell in the browser. It seems to contain the information I need for debugging this.

**Comment by barche** [comment]
OK, so I still can't figure it out. The problem is that `/opt/julia_packages/.julia/v0.5` is missing, but the 0.3 and 0.6 directories are there, even though I am not creating those anymore in the Dockerfiles as far as I can tell. The dates on these directories are also 17 September, while I would expect them to have the timestamp corresponding to image creation.

**Comment by barche** [comment]
Finally, it's solved. In case anyone else is bitten by this: the Julia packages are first extracted to `/jboxengine/data/packages/julia_packages/` and are reused as long as that directory is not removed.
